apps/client/src/components/Location/LocationFinder.js

```diff
@@ -83,10 +92,32 @@ const LocationFinder = (props) => {
   const houseNumberError = validate("houseNumber", houseNumber, true);
 
   const handleBlur = (e) => {
-    setTouched({ ...touched, [e.target.name]: true });
+    e.target.value && setTouched({ ...touched, [e.target.name]: true });
```

Comment:
This fixes the focus error: when entering the page and you un-focus the default input we used to get an error on that input "this field should be set". This is now a thing from the past.